test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,34 @@
+#include "TutorialServo.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1,
+                             float maxPulsewidthInMs = 2)
+    : m_servoPwmOut(servoPin),
+      m_servoRangeInDegrees(servoRangeInDegrees),
+      m_minPulsewidthInMs(minPulsewidthInMs) m_maxPulsewidthInMs(maxPulsewidthInMs) {
+  // period depends on the servo itself, check datasheet
+  servoPwnOut.period_ms(20);
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+  // assume degress are always < 360
+
+  // neglect the case where desire degree to set to is greater than the range
+  if (degrees > m_servoRangeInDegrees) return;
```

Comment:
if the the user specified `degrees` is greater than the servos range, it would be best to set it to the max position. Also what would happen if the `degrees` provided is negative?